bots/csr/src/main/java/org/openjdk/skara/bots/csr/CSRBot.java

```diff
@@ -102,6 +119,13 @@ public Collection<WorkItem> run(Path scratchPath) {
             var csr = csrOptional.get();
 
             log.info("Found CSR for " + describe(pr) + ". It has id " + csr.id());
```

Comment:
I reacted to this log message the other day. Could you shorten it like this while in the area?

```suggestion
            log.info("Found CSR " + csr.id() + " for " + describe(pr) + ");
```

Reply:
Fixed.